Cargo.toml

```diff
@@ -40,6 +40,7 @@ deflate = { version = "0.9.1", optional = true }
 sha1 = { version = "0.6.0", features = ["std"] }
 tempfile = "3.2.0"
 futures = "0.3.17"
+base64 = "=0.20.0-alpha.1"
```

Comment:
this is an alpha release, though the author suggests that the API and implementation are stable. I've pinned the version to mitigate any risk here.

see https://github.com/marshallpierce/rust-base64/issues/174#issuecomment-1032751831